### PR TITLE
Bxmsdoc 6038 master

### PR DESCRIPTION
**Pull request by mramendi**

### CODE REVIEW

File: doc-content/enterprise-only/openshift/githooks-prepare-proc.adoc [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
++
+. Create a configuration map (ConfigMap) or persistent volume with the files.
+** If the Git hooks consist of one or several fixed script files, use the `oc` command to create a configuration map. For example:
++
+[subs="attributes,verbatim,macros"]
+----
+oc create configmap git-hooks --from-file=post-commit=post-commit
+----
++
+** If the Git hooks consist of long files or depend on binaries, such as executable or KJAR files, use a persistent volume. You must create a persistent volume, create a persistent volume claim and associate the volume with the claim, and transfer files to the volume.
```
Comment (ruromero) [inline doc-content/enterprise-only/openshift/githooks-prepare-proc.adoc:27]: I think you meant JAR files
```suggestion
** If the Git hooks consist of long files or depend on binaries, such as executable or JAR files, use a persistent volume. You must create a persistent volume, create a persistent volume claim and associate the volume with the claim, and transfer files to the volume. 
```

File: doc-content/enterprise-only/openshift/githooks-prepare-proc.adoc [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+----
++
+** If the Git hooks consist of long files or depend on binaries, such as executable or KJAR files, use a persistent volume. You must create a persistent volume, create a persistent volume claim and associate the volume with the claim, and transfer files to the volume. 
++
+For instructions about persistent volumes and persistent volume claims, see https://access.redhat.com/documentation/en-us/openshift_container_platform/4.4/html/storage/index[Storage] in the {OPENSHIFT} documentation. For instructions about copying files onto a persistent volume, see https://blog.openshift.com/transferring-files-in-and-out-of-containers-in-openshift-part-3/[Transferring files in and out of containers].
++
+. If the Git hooks scripts must push commits into an upstream repository using SSH authentication, prepare a secret with the necessary files:
+.. Prepare the `id_rsa` file with a private key that matches a public key stored in the repository.
+.. Prepare the `known_hosts` file with the correct name, address, and public key for the repository.
+.. Create a secret with the two files using the `oc` command, for example:
```
Comment (ruromero) [inline doc-content/enterprise-only/openshift/githooks-prepare-proc.adoc:34]: I'd mention where these files are expected to be present i.e. where the volume should be mounted
`/home/jboss/.ssh`

File: doc-content/enterprise-only/openshift/customimage-con.adoc [hunk 3]
```diff
@@ -1,12 +1,11 @@
 [id='customimage-con_{context}']
-= Creating custom images for {CENTRAL} and {KIE_SERVER}
+= Creating custom images for {KIE_SERVER}
 
-You can create custom images to add files to the {CENTRAL} and {KIE_SERVER} deployments. You must then push the images to your own Docker registry. When deploying {PRODUCT}, you can configure the operator to use the custom images.
+You can create custom images to add files to {KIE_SERVER} deployments. You must then push the images to your own container registry. When deploying {PRODUCT}, you can configure the operator to use the custom images.
```
Comment (michelehaglund) [inline doc-content/enterprise-only/openshift/customimage-con.adoc:4]: I suggest to remove "then" from: You must then push the images to your own container registry.

File: doc-content/enterprise-only/openshift/operator-deploy-pim-proc.adoc [hunk 4]
```diff
@@ -1,7 +1,7 @@
 [id='operator-deploy-pim-proc']
 = Setting Process Instance Migration configuration for the environment
 
-You can use the operator to deploy the Process Instance Migration (PIM) service. You can use the PIM service to define the migration between two different process definitions, known as a migration plan. You can then apply the migration plan to the running process instances in a specific KIE Server. 
+You can use the operator to deploy the Process Instance Migration (PIM) service. You can use the PIM service to define the migration between two different process definitions, known as a migration plan. You can then apply the migration plan to the running process instances in a specific {KIE_SERVER}.
```
Comment (michelehaglund) [inline doc-content/enterprise-only/openshift/operator-deploy-pim-proc.adoc:4]: For minimalism, I suggest to remove "then" from: You can then apply the migration plan to the running process instances in a specific {KIE_SERVER}.

File: doc-content/enterprise-only/openshift/operator-modify-proc.adoc [hunk 5]
```diff
@@ -10,7 +10,8 @@ To modify the environment, you must modify the YAML description of the environme
 . In the OpenShift Web console navigation panel, select *Catalog -> Installed operators* or *Operators -> Installed operators*.
 . Find the `Business Automation` operator line in the table and click `KieApp` in the line. Information about the environments that you deployed using this operator is displayed.
 . Click the name of a deployed environment. 
-. Select the *YAML* tab. A YAML source is displayed.
+. Select the *YAML* tab. A YAML source is displayed.
```
Comment (michelehaglund) [inline doc-content/enterprise-only/openshift/operator-modify-proc.adoc:13]: Probably correct as is, but is it "A YAML.." or "The YAML.."?

File: doc-content/enterprise-only/openshift/githooks-prepare-proc.adoc [hunk 6]
```diff
@@ -0,0 +1,44 @@
+[id='githooks-prepare-proc-{context}']
+= Preparing Git hooks 
+
+In an authoring environment you can use Git hooks to execute custom operations when the source code of a project in {CENTRAL} is changed. The typical use of Git hooks is interaction with an upstream repository.
```
Comment (michelehaglund) [inline doc-content/enterprise-only/openshift/githooks-prepare-proc.adoc:4]: "The typical use of Git hooks is interaction with an upstream repository. "

Would it be correct to say: The typical use of Git hooks is for interaction with an upstream repository.

File: doc-content/enterprise-only/openshift/githooks-prepare-proc.adoc [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+
+In an authoring environment you can use Git hooks to execute custom operations when the source code of a project in {CENTRAL} is changed. The typical use of Git hooks is interaction with an upstream repository. 
+
+To enable Git hooks to interact with an upstream repository using SSH authentication, you must also provide a secret key and a known hosts file for authentication with the repository.
+
+Skip this procedure if you do not want to configure Git hooks.
+
+.Procedure
+
+. Create the Git hooks files. For instructions, see the https://git-scm.com/docs/githooks[Git hooks reference documentation].
```
Comment (michelehaglund) [inline doc-content/enterprise-only/openshift/githooks-prepare-proc.adoc:12]: For minimalism, I suggest: Create the Git hooks files. See https://git-scm.com/docs/githooks[Git hooks reference documentation].

File: doc-content/enterprise-only/openshift/githooks-prepare-proc.adoc [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+[subs="attributes,verbatim,macros"]
+----
+oc create configmap git-hooks --from-file=post-commit=post-commit
+----
++
+** If the Git hooks consist of long files or depend on binaries, such as executable or JAR files, use a persistent volume. You must create a persistent volume, create a persistent volume claim and associate the volume with the claim, and transfer files to the volume. 
++
+For instructions about persistent volumes and persistent volume claims, see https://access.redhat.com/documentation/en-us/openshift_container_platform/4.4/html/storage/index[Storage] in the {OPENSHIFT} documentation. For instructions about copying files onto a persistent volume, see https://blog.openshift.com/transferring-files-in-and-out-of-containers-in-openshift-part-3/[Transferring files in and out of containers].
++
+. If the Git hooks scripts must push commits into an upstream repository using SSH authentication, prepare a secret with the necessary files:
```
Comment (michelehaglund) [inline doc-content/enterprise-only/openshift/githooks-prepare-proc.adoc:31]: Should this say: prepare a secret key..

Comment (mramendi) [reply doc-content/enterprise-only/openshift/githooks-prepare-proc.adoc:31]: A "secret" is an OpenShift entity, so the wording is correct.

File: doc-content/enterprise-only/openshift/githooks-prepare-proc.adoc (hunk 7 above)
Comment (mramendi) [reply doc-content/enterprise-only/openshift/githooks-prepare-proc.adoc:12]: In other places we tend to write "for instructions" or "for more information", and for all I know a mere "see..." without clarification is discouraged? I don't think there are written guidelines though

File: doc-content/enterprise-only/openshift/operator-modify-proc.adoc (hunk 5 above)
Comment (mramendi) [reply doc-content/enterprise-only/openshift/operator-modify-proc.adoc:13]: I think I'd need to add a description of the exact meaning of the source with "the", and I'm not sure I have teh resources to get this formulation exactly right.

**Pull request merged**